packages/shared/src/utils/constants/contract.ts

```diff
@@ -392,95 +402,95 @@ export const getUnsupportedContracts = () =>
             name: localize('Spread Down'),
             position: 'bottom',
         },
+        RUNHIGH: {
+            name: localize('Only Ups'),
+            position: 'top',
```

Comment:
@kate-deriv  we do we add these in this PR? :)

Reply:
Will remove, thank you